lib/compat/wordpress-6.3/html-api/class-gutenberg-html-tag-processor-6-3.php

```diff
@@ -2282,11 +2282,12 @@ private function matches() {
 			 * See https://html.spec.whatwg.org/#attributes-3
 			 * See https://html.spec.whatwg.org/#space-separated-tokens
 			 */
-			while (
-				// phpcs:ignore WordPress.CodeAnalysis.AssignmentInCondition.FoundInWhileCondition
-				false !== ( $class_at = strpos( $this->html, $this->sought_class_name, $class_at ) ) &&
-				$class_at < $class_end
-			) {
+			while ( true ) {
```

Comment:
Same comment as per https://github.com/WordPress/gutenberg/pull/53866/files#diff-9b0b96f086429280a9f988226c77db353802dadbf18552cc9cf55db9da1a04aeR2254 / `while` condition in `lib/compat/wordpress-6.2/html-api/class-wp-html-tag-processor.php` file.